.github/workflows/ci.yml

```diff
@@ -162,6 +156,58 @@ jobs:
         run: |
           sudo ethtool -K eth0 tx off
           sudo ethtool -K eth0 rx off
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: set up go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ env.go-version }}
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/go/pkg/mod
+            ~/.cache/go-build
+          key: ${{ runner.os }}-cache-go-${{ env.go-version }}-unittests-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-cache-go-${{ env.go-version }}-unittests-
+            ${{ runner.os }}-cache-go-${{ env.go-version }}-
+      - name: setup env
+        run: make install
+      - name: Clear test cache
+        run: make clear-test-cache
+      - name: unit tests
+        timeout-minutes: 25
+        env:
+          GOTESTSUM_FORMAT: standard-verbose
+          GOTESTSUM_JUNITFILE: unit-tests.xml
+        run: make test
+        continue-on-error: ${{ matrix.allow-failure == 'true' }}
+      - name: Publish Test Report
+        uses: mikepenz/action-junit-report@v3
+        # always run even if the previous step fails
+        if: always()
+        with:
+          report_paths: "**/unit-tests.xml"
+          annotate_only: true
+          fail_on_failure: ${{ matrix.allow-failure == 'false' }}
+
+
+  unittests-slower:
```

Comment:
I think `unittests-non-linux` or similar would be a more expressive name 🙂

Reply:
Well there could be m1 also coming etc. I kinda left that open "just in case".